moveit_core/constraint_samplers/test/test_constraint_samplers.cpp

```diff
@@ -673,8 +673,8 @@ TEST_F(LoadPlanningModelsPr2, PoseConstraintSamplerManager)
     if (s->sample(ks, ks_const, 1))
       succ++;
   }
-  ROS_INFO("Success rate for IK Constraint Sampler with position & orientation constraints for one arm: %lf",
-           (double)succ / (double)NT);
+  // ROS_INFO("Success rate for IK Constraint Sampler with position & orientation constraints for one arm: %lf",
```

Comment:
This info should be printed, either by a logger or using std::cout.

Reply:
https://github.com/AcutronicRobotics/moveit2/commit/05ad105ae4a5b23a1f660187d42790479924c911